dev-support/mini-submarine/build_mini-submarine.sh

```diff
@@ -13,10 +13,16 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
+set -e
 hadoop_v=2.9.2
 spark_v=2.4.4
-submarine_v=0.3.0-SNAPSHOT
+
+submarine_v=$submarine_version
```

Comment:
submarine_v =${submarine_version:-"0.3.0-SNAPSHOT"}
This way, So you can set default values.